src/analysis/Analysis.cpp

```diff
@@ -278,7 +267,7 @@ void Analysis::calc_RMS_error(real sum_u, real sum_p, real sum_T) {
 /// \param  u     x-velocity field
 /// \param  dt      time step size
 // ***************************************************************************************
-bool Analysis::check_time_step_VN(Field *u, real dt) {
+bool Analysis::check_time_step_VN(Field &u, real dt) {
```

Comment:
Field u isn't used in this method. We can remove it, right?

Reply:
done in 767dab7